docs/en/observability/profiling-get-started.asciidoc

```diff
@@ -1,128 +1,166 @@
+[[profiling-set-up-on-cloud]]
+== Set up Universal Profiling on an {ecloud} deployment
+
+To set up Universal Profiling on your {ecloud} deployment, you need to:
+
+- Enable the Universal Profiling app in Kibana
+- Configure data ingestion
+
+[discrete]
+[[profiling-enable-kibana]]
+=== Enable Universal Profiling in Kibana
+
+You'll receive instructions on enabling Universal Profiling at the email you used in your https://docs.google.com/forms/d/e/1FAIpQLSd-SWVgvhO7Z_jAfaV9_bFGa0dUZPuX0JORzPGS8SDP7G-dVQ/viewform[sign-up form].
+
+Once enabled, Universal Profiling in Kibana is located under *Observability* in the navigation menu.
+
+[discrete]
+[[profiling-configure-data-ingestion]]
+=== Configure data ingestion
+
+After enabling Universal Profiling on your deployment for the first time, select any subheading under **Universal Profiling** in the navigation menu to open the following page:
+
+[role="screenshot"]
+image::images/profiling-setup-popup.png[]
+
+Click *Set up Universal Profiling* to configure data ingestion.
+
+NOTE: To configure data ingestion, you need elevated privileges, typically the `elastic` user.
+
+If you're upgrading from a previous version with Universal Profiling enabled, see the <<profiling-upgrade,upgrade guide>>.
+
+IMPORTANT: When upgrading, you must remove all existing profiling data.
+We still recommend upgrading as the latest version contains several improvements and new features.
+
+[discrete]
+[[profiling-install-host-agent]]
+== Install the host-agent
+
+The host-agent profiles your fleet. You need to install and configure it on every machine that you want to profile.
+The host-agent needs  `root` / `CAP_SYS_ADMIN` privileges to run.
+
+After clicking *Set up Universal Profiling* in the previous step, you'll see the instructions for installing the host-agent.
+You can also find these instructions by clicking the *Add data* button in the top-right corner of the page.
+
+The following is an example of the provided instructions for {k8s}:
+
+[role="screenshot"]
+image::images/profiling-k8s-hostagent.png[]
+
+[discrete]
+[[profiling-agent-config-notes]]
+=== Host-agent configuration notes
+
+Consider the following when configuring your host-agent:
+
+* The instructions in Kibana work well for testing environments. For production environments, we recommend setting an immutable version.
+
+* The host-agent versioning scheme is **not aligned with the {stack} version scheme**.
+
+* The OS packages downloaded from `releases.prodfiler.com` have a version in their file name.
+
+* You can find a list of container image versions in the
+https://container-library.elastic.co/r/observability/profiling-agent[Elastic container library repository].
+
+* For {k8s} deployments, the Helm chart version is already used to configure the same container image, unless
+overwritten with the `version` parameter in the Helm values file.
+
+[discrete]
+[[profiling-install-integration]]
+== Install the Universal Profiling Agent integration
+
```

Comment:
I often see a headline without text as a lost opportunity. Can we add some text here?
E.g. we could point out the multiple ways our solution can be installed.
```
{kib} -> > **Universal Profiliing** -> **Add data**
```